build/helm/keylime/values.yaml

```diff
@@ -161,11 +161,15 @@ global:
       # Default image is the from quay, default tag is tag the chart appVersion
       image:
         repository: quay.io/keylime/keylime_tenant
+      # Default image pull policy
+        pullPolicy: IfNotPresent
```

Comment:
Wondering whether "always" might not be a better default policy. Especially for people who like "latest" or do quick development. 
But certainly having any default pull policy is better than none.